apps/stats/stats-1.py

```diff
@@ -89,19 +89,12 @@ def __init__( self ) :
 				),
 
 				IECore.IntParameter(
-					name = "mostFrequentlyHashed",
```

Comment:
Yeah, you're totally right - it's been so long since a release we haven't even got the first pass of the PerformanceMonitor out have we? I'll remove those bogus comments and the label then...